custom/enikshay/tasks.py

```diff
@@ -201,13 +204,16 @@ def _get_open_episode_cases(self, case_ids):
         case_accessor = CaseAccessors(self.domain)
         episode_cases = case_accessor.iter_cases(case_ids)
         for episode_case in episode_cases:
+            if episode_case.get_case_property('is_active') != 'true':
+                continue
+
             # if this episode is part of a deleted or archived person, don't update
             try:
                 person_case = get_person_case_from_episode(self.domain, episode_case.case_id)
             except ENikshayCaseNotFound:
                 continue
 
-            if person_case.owner_id == ARCHIVED_CASE_OWNER_ID:
+            if person_case.owner_id in [ARCHIVED_CASE_OWNER_ID, INVALID_CASE_OWNER_ID]:
```

Comment:
Same comment as before - depending on how this utility is used, we may want to include archived cases

Reply:
I am not sure about the enikshay task. Will defer to @proteusvacuum if he knows why archived cases were ignored here.

Reply:
this filters out archived cases so that we don't update them every night. If a case becomes unarchived, it will start getting updated again.

Reply:
@mkangia 

We may want to rename the function so its clearer what it does or provide an option to include or not include archived cases?

Reply:
This is only used in the task (and is a private method of the task, so shouldn't be used by anything other than that).